server/taskflows/hpccloud/taskflow/paraview.py

```diff
@@ -87,26 +65,14 @@ def start(self, *args, **kwargs):
         kwargs['image_spec'] = image_spec
```

Comment:
This seems redundant with the line above. I assume this statement should only be called once but should it be in the if block or not?